galaxy_ng/app/utils/pulp_rbac.py

```diff
@@ -0,0 +1,45 @@
+from collections import defaultdict
+
+from django.db.models import Q
+from django.contrib.auth.models import Permission
+from django.contrib.contenttypes.models import ContentType
+
+from pulpcore.plugin.models.role import GroupRole
+from pulpcore.plugin.models.role import UserRole
+
+
+def get_users_with_perms_attached_roles(
+    obj,
+    with_group_users=True,
+    only_with_perms_in=None,
+    include_domain_permissions=True,
+    include_model_permissions=True,
+    for_concrete_model=False,
+):
+    # DELETE ONCE https://github.com/pulp/pulpcore/pull/4479 IS RELEASED AND BUMPED IN GALAXY_NG
```

Comment:
Can be deleted now? looks like https://github.com/pulp/pulpcore/issues/4478 was released in 3.28.16.